resources/es.ftl

```diff
@@ -0,0 +1,59 @@
+auth = Por favor usa el siguiente enlace para autenticarte en tu cuenta Wikimedia: [Authenticate]({$url})
+
+auth_exists_in_server = Actualmente estás autenticado en este servidor. No es necesario autenticarse de nuevo.
+
+auth_to_server = Estás identificado como [{$name}](<{$url}>). ¿Quieres autenticarte de nuevo en este servidor?
+yes = sí
+no = No
```

Comment:
Can si be capitalized? this is the text on buttons so consistency would be good.